README.md

```diff
@@ -100,24 +88,24 @@ Now the `Klusterlet` operand will start the pods for `registration` and `work`.
 We are going to take an insecure approach to import the _managed cluster_ to the _hub_ because we cannot create `ServiceAccount` tokens in KinD clusters. So let's capture the `kind-hub` `$KUBECONFIG` and create a `Secret` with it on the _managed clusters_.
 
 ```bash
-$ export KUBECONFIG=kind-hub.kubecfg
-$ kind export kubeconfig --name=hub
-
-./generate-bootstrap-kubeconfig.sh
-
+$ kubectl config use kind-hub
+$ ./generate-bootstrap-kubeconfig.sh
+$ kubectl config use kind-cluster
 $ kubectl create secret generic bootstrap-hub-kubeconfig \
     -n open-cluster-management-agent \
-    --from-file=kubeconfig=bootstrap-hub.kubecfg
+    --from-file=kubeconfig=bootstrap-hub.kubeconfig
 ```
 
 Approve the `CertificateSigningRequest` on the _hub_.
 
 ```bash
+$ kubectl config use kind-hub
 $ kubectl get csr
 NAME                 AGE     SIGNERNAME                            REQUESTOR          CONDITION
 kind-cluster-mgtdz   8m53s   kubernetes.io/kube-apiserver-client   kubernetes-admin   Pending
 
 $ kubectl certificate approve kind-cluster-mgtdz
+$ kubectl patch managedcluster kind-cluster -p='{"spec":{"hubAcceptsClient":true}}' --type=merge
```

Comment:
I broke up these steps in the flow to explain what the different commands meant for new users.